core/os/os.cpp

```diff
@@ -37,7 +37,9 @@
 #include "core/os/midi_driver.h"
 #include "core/version_generated.gen.h"
 
+#include <limits.h>
 #include <stdarg.h>
+#include <unistd.h>
```

Comment:
Is there `unistd.h` on Windows, maybe in MinGW, but likely not in MSVC?

The same `gethostname` is available in the `winsock.h` (and length probably should be hard-coded to 256, since there's no `HOST_NAME_MAX` variable or `sysconf`).

https://docs.microsoft.com/en-us/windows/win32/api/winsock/nf-winsock-gethostname